### PR TITLE
test: add integration tests

### PR DESCRIPTION
**Pull request by ahmad-ibra**

### CODE REVIEW

File: internal/controller/suite_test.go [hunk 1]
```diff
@@ -71,10 +120,59 @@ var _ = BeforeSuite(func() {
...
+	Expect(err).NotTo(HaveOccurred(), "failed to create validator namespace")
+
+	k8sManager, err := ctrl.NewManager(cfg, ctrl.Options{
+		Scheme: scheme.Scheme,
+	})
+	Expect(err).ToNot(HaveOccurred(), "failed to init manager")
+
+	err = (&OciValidatorReconciler{
+		Client: k8sManager.GetClient(),
+		Log:    ctrl.Log.WithName("controllers").WithName("AwsValidator"),
```
Comment (TylerGillson) [inline internal/controller/suite_test.go:134]: ⛏️ AwsValidator -> OciValidator

File: internal/controller/suite_test.go [hunk 2]
```diff
@@ -71,10 +120,59 @@ var _ = BeforeSuite(func() {
...
+		Scheme: k8sManager.GetScheme(),
+	}).SetupWithManager(k8sManager)
+	Expect(err).ToNot(HaveOccurred(), "failed to start AwsValidator controller")
```
Comment (TylerGillson) [inline internal/controller/suite_test.go:137]: ⛏️ AwsValidator -> OciValidator

Comment (ahmad-ibra) [reply internal/controller/suite_test.go:137]: thanks for catching that!

**Review by TylerGillson — APPROVED** [review]
lgtm

**Pull request merged**